docs/concepts/privacy-and-security.md

```diff
@@ -251,9 +251,9 @@ The management and enforcement of Office settings is done with group policy sett
 
 | Setting name | Description |
 |--------------|-------------|
-| Allow Unsecure web add-ins and Catalogs | Allows users to run non-secure add-ins, which are add-ins that have webpage or catalog locations that are not SSL-secured (https://) and are not in users' Internet zones. |
-| Block Web Add-ins | Allows you to prevent users from using web add-ins. |
-| Block the Office Store |  Allows you to prevent users from using or inserting web add-ins that come from the Office Store. |
+| Allow Unsecure web add-ins and Catalogs | Allows users to run non-secure Office Add-ins, which are Office Add-ins that have webpage or catalog locations that are not SSL-secured (https://) and are not in users' Internet zones. |
+| Block Web Add-ins | Allows you to prevent users from running Office Add-ins using web technologies. |
```

Comment:
Suggestion to make this not sound like we're using web technologies to block the add-ins.
```suggestion
| Block Web Add-ins | Allows you to prevent users from running Office Add-ins that use web technologies. |
```